### ISSUE TITLE
show In-line PR comments besides code

### ISSUE THREAD

**Issue by chuan29812** [pwntester/octo.nvim#715]
## Issue Description

Type: *feature request*

### Describe what happened (or what feature you want)
JetBrain recently added this new feature where if you hooked up your github account to the editor, you can now see inlined comment within the editor right below your code if it has an associated PR.
![in-line pr comment](https://github.com/user-attachments/assets/6f79139b-1fed-48aa-bc51-8feb0880d1dd)

I believe this is not something that can currently be accomplished.
I wonder if would be in the scope of this plugin to add something like that

**Comment by wd60622** [comment]
Related to "Edit and review GitHub issues and pull requests" so think it fits. Would be great addition!
Are there any plugins that already support it?

**Comment by chuan29812** [comment]
@wd60622 sadly no, think they just added this to Clion like 2 weeks ago, and rubymine last week. I posted on the neovim subreddit, and no one seems to know something like this that currently exists. 

If im interested in building this, would I need a definite yes or can I just start creating a PR and see where that goes?

**Comment by wd60622** [comment]
> @wd60622 sadly no, think they just added this to Clion like 2 weeks ago, and rubymine last week. I posted on the neovim subreddit, and no one seems to know something like this that currently exists.
> 
> If im interested in building this, would I need a definite yes or can I just start creating a PR and see where that goes?

Yeah, sure! Give it a try. Feel free to open even a draft if you need any eyes or people trying out
